Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -3692,6 +3692,12 @@ void RenderLayerBacking::paintDebugOverlays(const GraphicsLayer* graphicsLayer,
         context.setStrokeThickness(1);
 
         for (const auto& region : eventRegion.interactionRegions()) {
+            
+            if (region.type == InteractionRegion::Type::Occlusion)
+                context.setStrokeColor(Color::red);
+            else
+                context.setStrokeColor(Color::green);
```

Comment:
Since `Color::green` is always being set in `else`, you can remove the `context.setStrokeColor(Color::green);` on line 3691 above.